lonboard/_map.py

```diff
@@ -151,6 +131,35 @@ def on_click(self, callback: Callable, *, remove: bool = False) -> None:
     _esm = bundler_output_dir / "index.js"
     _css = bundler_output_dir / "index.css"
 
+    _has_click_handlers = t.Bool(default_value=False, allow_none=False).tag(sync=True)
```

Comment:
This diff is from moving `view_state` lower so that it's immediately after the new `views` attribute; nothing was changed except for adding `views`